prod/monitoring/installation/values-workload.yaml

```diff
@@ -0,0 +1,72 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: values-workload
+  namespace: flux-system
+data:
+  values.yaml: |
+    thanosStorage:
+      secret:
+        name: thanos-objstore-config
+        key: objstore.yml
+
+    thanos:
+      fullnameOverride: thanos
+      existingObjstoreSecret: thanos-objstore-config
+      bucketweb:
+        enabled: true
+      storegateway:
+        enabled: true
+      compactor:
+        enabled: true
+        persistence:
+          size: 20Gi
+      query:
+        replicaCount: 2
+        replicaLabel: [prometheus_replica]
+    ## FIXME: Thanos query is temporary exposed via (unsecure) NodePort, see https://github.com/SovereignCloudStack/k8s-observability/issues/32
+    ##   Thanos query should be exposed via ingress, see below commented section
+        serviceGrpc:
+          type: NodePort
+          nodePorts:
+            grpc: "31050"
+
+    containerLabel: &containerLabel
```

Comment:
Where is this used? Do you miss the `dnation-kubernetes-monitoring` section here? See https://github.com/SovereignCloudStack/k8s-observability/blob/main/registry/workload/values-workload.yaml#L39

Reply:
`containerLabel` field is unnecessary for now and can be deleted.

Reply:
And what about `dnation-kubernetes-monitoring` section?

Reply:
Ok, I see. Thanks